setup.py

```diff
@@ -0,0 +1,3 @@
+from setuptools import setup
+
+setup()
```

Comment:
https://setuptools.pypa.io/en/latest/setuptools.html

I don't think this is necessary from `40.9.0`.

I'd rather include a `[build-system]` section than add an additional file.

Reply:
I did not manage to do an editable install or launch pytest in terminal, I'm going to try an up to date setuptools.

Reply:
```
      Successfully uninstalled setuptools-44.0.0
Successfully installed setuptools-60.2.0
```

```
main  pip3 install -e .
ERROR: File "setup.py" not found. Directory cannot be installed in editable mode: /home/pierre/pydocstringformatter
(A "pyproject.toml" file was found, but editable mode currently requires a setup.py based build.)
```

Reply:
Are you sure your `pip3` installs to PATH? 

```console
pip install -e .
Obtaining file:///Users/daniel/DocumentenLaptop/Programming/Github/DanielNoord/docstringformatter
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied: tomli>=1.1.0 in /Users/daniel/.pyenv/versions/3.10.0/envs/docstring-3.10.0/lib/python3.10/site-packages (from pydocstringformatter==0.3.0+dev) (1.2.3)
Installing collected packages: pydocstringformatter
  Attempting uninstall: pydocstringformatter
    Found existing installation: pydocstringformatter 0.2.0+dev
    Uninstalling pydocstringformatter-0.2.0+dev:
      Successfully uninstalled pydocstringformatter-0.2.0+dev
  Running setup.py develop for pydocstringformatter
Successfully installed pydocstringformatter-0.3.0+dev
WARNING: You are using pip version 21.2.3; however, version 21.3.1 is available.
You should consider upgrading via the '/Users/daniel/.pyenv/versions/3.10.0/envs/docstring-3.10.0/bin/python3.10 -m pip install --upgrade pip' command.
```

It works fine for me..

Reply:
I can launch the test from my IDE, it's mouse heavy but it works. I'll resolve this

Reply:
I've just added the `build-system` to `pyproject.toml`. Perhaps it works now?

What does `which setuptools` return?